CH-662 bug - changed validation

NatalliaBulyha opened the pull request:
jira[If you enter an invalid "email" and "password", you will receive an incorrect response from the server] https://cozyhometeam.atlassian.net/browse/CH-662?atlOrigin=eyJpIjoiOTU3NzQwMWU0Yjg5NDJhNzg0YWY1MTA5ZmM2OTZmYzkiLCJwIjoiaiJ9

Commit:
CH-662 bug - changed validation

jira[If you enter an invalid "email" and "password", you will receive an incorrect response from the server] https://cozyhometeam.atlassian.net/browse/CH-662?atlOrigin=eyJpIjoiOTU3NzQwMWU0Yjg5NDJhNzg0YWY1MTA5ZmM2OTZmYzkiLCJwIjoiaiJ9

DariaBoo commented on src/main/java/com/cozyhome/onlineshop/dto/auth/LoginRequest.java:
```diff
@@ -17,6 +16,5 @@ public class LoginRequest {
 	@ValidEmail
 	private String username;
```
can we rename it to 'email'?  because it's a little bit confused: the different names for validation and field

NatalliaBulyha replied:
done

Commit:
CH-662 bug corrections as noted

jira[If you enter an invalid "email" and "password", you will receive an incorrect response from the server] https://cozyhometeam.atlassian.net/browse/CH-662?atlOrigin=eyJpIjoiOTU3NzQwMWU0Yjg5NDJhNzg0YWY1MTA5ZmM2OTZmYzkiLCJwIjoiaiJ9

Commit:
Merge remote-tracking branch 'origin/main' into CH-662-bug

# Conflicts:
#	src/main/java/com/cozyhome/onlineshop/userservice/security/service/impl/UserServiceImpl.java
